src/tests/JIT/Directed/RVAInit/simplearg.il

```diff
@@ -1,6 +1,5 @@
-
-//  Microsoft (R) .NET Framework IL Disassembler.  Version 4.7.3071.0
-//  Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is it intentional that you aren't adding the license header to the src/mono/mono/tests/* files?

Reply:
I do not know where these files came from originally. Adding the simple license headers may be actually wrong for them.

Also, nearly all files under `src/mono/mono/tests/*` are missing the license headers so it is also for consistency.